packages/cozy-client/src/assets/qualifications.json

```diff
@@ -366,25 +389,66 @@
     {
       "label": "receipt",
       "purpose": "report"
+    },
+    {
+      "label": "payment_proof_family_allowance",
+      "purpose": "attestation",
+      "sourceCategory": "gov",
+      "sourceSubCategory": "tax",
+      "subjects": ["subvention"]
+    },
+    {
+      "label": "school_attendance_certificate",
+      "purpose": "attestation",
+      "sourceCategory": "education",
+      "subjects": ["identity", "right", "education"]
+    },
+    {
+      "label": "unfit_for_habitation_declaration",
+      "purpose": "attestation",
+      "sourceCategory": "gov",
+      "sourceSubCategory": "real_estate",
+      "subjects": ["compliance"]
+    },
+    {
+      "label": "accommodation_proof",
+      "purpose": "attestation",
+      "sourceCategory": "individual",
+      "subjects": ["address"]
+    },
+    {
+      "label": "citizen_registration_certificate",
+      "purpose": "attestation",
+      "sourceCategory": "gov",
+      "sourceSubCategory": "civil_registration",
+      "subjects": ["achievement","education"]
+    },
+    {
+      "label": "personal_sporting_licence",
+      "purpose": "attestation",
+      "sourceCategory": "association",
+      "sourceSubCategory": "sport",
+      "subjects": ["permit", "invoice"]
```

Comment:
I don't think a sport license is an invoice?

Reply:
Indeed not, but the invoice is the subject of the attestation. this licence attests I paid for the permit...for this reason I would keep "invoice" in the subject.
But "association" should be replaced with "gov", because a personal sport licence is issued by the ministry of sports.

Reply:
>  this licence attests I paid for the permit...

The license proves that I belong to a sport club, the payment does not seem very relevant to me, here. I guess *technically* you could have a sport association that does not ask payment for its members :)